docs/ci/codemagic.mdx

```diff
@@ -99,6 +100,9 @@ workflows:
       groups:
         # Exports the SHOREBIRD_TOKEN environment variable
         - shorebird
+      vars:
+        # Skip confirmation prompts
+        CONTINUOUS_INTEGRATION: 'true'
```

Comment:
Minor but might want to rename to CI for consistency with other setups (e.g flutter, GitHub, GitLab, etc). Also if we did that users wouldn’t need to specify this value since it’s automatically set.

Reply:
Too slow :) Take a look at the updated comment and code